src/introduction-to-unified.md

```diff
@@ -0,0 +1,215 @@
+
+#### Tools
+
+*   [mdx][mdx] — JSX in Markdown.
+*   [vfile][vfile] — Virtual file format for text processing.
+
+#### Specifications
+
+*   [unist][unist] — Universal Syntax Tree.
+*   [mdast][mdast] — Markdown Abstract Syntax Tree format.
+*   [hast][hast] — HTML Abstract Syntax Tree format.
+*   [nlcst][nlcst] — Natural Language Concrete Syntax Tree format.
+
+* * *
+
+Don’t worry, we’ll get to how all of this comes together in unified.
+But if you are already feeling adventurous; you can go directly to
+[“Using unified”](<>) or [“How to get started with plugins”](<>).
+
+* * *
+
+### How does it all come together
+
+unified can integrate with the file-system through
+[unified-engine][unified-engine].  On top of that, CLI apps can be created with
+[unified-args][unified-args], Gulp plugins with
+[unified-engine-gulp][unified-engine-gulp], and Atom Linters with
+[unified-engine-atom][unified-engine-atom].
+
+But how do these specifications, tools, and processors fit together?
+**Essentially, unified is a three part act**:
+
+1.  **Parse.**  Whether your input is natural text, HTML, or markdown
+    — it needs to be parsed to a universal workable format.
+    The processors (e.g [remark][remark]) are responsible for doing just that.
+    The specifications (e.g [mdast][mdast] for remark) make sure these formats
+    are enforced standards.
+2.  **Transform.**  This is where the magic happens.  Plugins are able to easily
+    hook into this phase and add, change, or remove anything that it’s given.
+    Thanks to unified’s pipeline, you can compose plugins and define the order
+    they run in.
+3.  **Stringify.**  When all plugins have ran, the final step is to take the
+    (adjusted) representation and transform it back to its original format
+    (or to a different format!).
+
+#### Bridging formats
+
+What makes unified unique is that it can continuously switch between formats,
+markdown to HTML for instance, in the same process.
+This allows for even more powerful compositions.
+
+Bridging formats can be done with the following plugins:
+
+*   [remark-rehype][remark-rehype] — markdown to HTML.
+*   [rehype-remark][rehype-remark] — HTML to markdown.
+
+### Use cases
+
+**Whenever you think about processing content — you can think of unified**.
+Unified makes it easier for developers to develop by providing the means to
+quickly iterate, manipulate, and create content.
+
+> A large part of MDX’s success has been leveraging the unified and remark
+> ecosystem.  I was able to get a prototype working in a few hours because
+> I didn’t have to worry about markdown parsing: remark gave it to me for free.
+> It provided the primitives to build on.
+>
+> — [John Otander][john], author of [mdx-js/mdx][mdx]
+
+To further speak to one’s imagination, here are the more common plugins used in
+the unified pipeline to do interesting things:
+
+*   [remark-toc][remark-toc]  —  generate a table of contents.
```

Comment:
spacing around the list description dash is sometimes two, sometimes one space, sometimes nonexistent